### PR TITLE
Feature/ab select variables

### PR DESCRIPTION
**Pull request by annagav**
Based on #371. Please review #371 first. 

Limit the selection of variables to 3 experimental variables. When 3 are selected it disables the rest.

### CODE REVIEW

**Comment by giocalitri** [comment]
if this is based on #371, what is the commit for this PR only?

**Comment by giocalitri** [comment]
please rename the migration of this PR to something meaningful

**Comment by giocalitri** [comment]
A couple of minor comment and the migration rename.

**Comment by annagav** [comment]
Ready for functional test.

**Comment by annagav** [comment]
Ready!

File: instructor/common.py [hunk 1]
```diff
@@ -234,17 +234,28 @@ def course_modify(request):
...
+            num_variables = 0
+            for field in var_fields:
+                if getattr(assignment, field):
+                    num_variables += 1
+            if num_variables <= max_num_of_vars:
+                form.save()
+            if 'continue' in request.POST:
+                return redirect("common_assignments_edit_treatments")
+
+    assignment = models.Assignment.objects.get(id=assignment_id)
```
Comment (giocalitri) [inline instructor/common.py:273]: please add a comment before this line to say that you are going to refresh the object from the DB.

**Comment by giocalitri** [comment]
The code is fine: you need to understand this conflict.

Than you can merge

**Pull request merged**